mlflow/utils/credentials.py

```diff
@@ -41,3 +46,129 @@ def read_mlflow_creds() -> MlflowCreds:
         username=username_env or username_file,
         password=password_env or password_file,
     )
+
+
+def login(backend="databricks"):
+    """Configure MLflow server authentication and connect MLflow to tracking server.
+
+    This method provides a simple way to connect MLflow to its tracking server. Currently only
+    Databricks tracking server is supported. Users will be prompted to enter the credentials if no
+    existing Databricks profile is found, and the credentials will be saved to `~/.databrickscfg`.
+
+    Args:
+        backend: string, the backend of the tracking server. Currently only "databricks" is
+            supported.
+
+    .. code-block:: python
+        :caption: Example
+
+        import mlflow
+
+        mlflow.login()
+        with mlflow.start_run() as run:
+            mlflow.log_param("p", 0)
+    """
+    if backend == "databricks":
+        _databricks_login()
+    else:
+        raise ValueError(
+            f"Currently only 'databricks' backend is supported, received `backend={backend}`."
+        )
+
+
+def _check_databricks_auth():
+    # Check if databricks credentials are set.
+    from databricks.sdk import WorkspaceClient
```

Comment:
What is a user doesn't have `databricks.sdk` installed?

Reply:
I think we can add `databricks-sdk` to the required deps list? Currently `databricks` is required, so I think it's okay

Reply:
It's fairly lightweight at 297.9kB. However, that being said, this might be a bit odd for OSS MLflow users. Can we just do an optimistic import check here and raise if the SDK is not installed?
I imagine that a lot of OSS users have no desire to have the SDK installed in their environment if they're not using Databricks at all.

Reply:
makes sense!